docs/environment_creation_from_scratch.md

```diff
@@ -1,54 +1,112 @@
 When setting up a new environment:
-1. DNS records will need to be created by DLUHC for certificate validation and to point at the cloudfront distribution and app load balancer
-1. Secret values will need to be filled in manually
-1. The meta account will need updating with additional roles access to the ecr, since this is currently manual
-1. A new deployment pipeline will need to be setup
+1. DNS records will need to be added by DLUHC for certificate validation and to point at the cloudfront distribution and 
+app load balancer
+2. Secret values will need to be filled in manually
+3. The meta account will need updating with additional roles access to the ecr, since this is currently manual
+4. A new deployment pipeline will need to be setup
+
+See the process below for detailed steps.
 
 ## The process
 
+### Create a new terraform folder
+
+If the environment hasn't been created / defined before:
+
+In the terraform folder, add a new sub-folder with the name of the environment.
+
+Add a `main.tf` file to the folder, and define any infrastructure here (e.g. modules to use). You may also add 
+`variables.tf` and `outputs.tf` if necessary.
+
+Ensure for the backend block in `main.tf` (see the equivalent in other environment folders for examples), you define a 
+unique key to store the infrastructure state in (i.e. the full name and extension of the terraform state file). 
+
+Once complete, while `cd`'d into the environment folder, run the following command to prepare terraform:
+
+```terraform init```
+
 ### Run an initial apply
 
-```terraform apply -target="module.networking" -target="module.front_door" -target="module.application_secrets" -var="initial_create=true"```
+Run the command below to create partial infrastructure. 
 
-(Need the networking explicitly otherwise we get an error creating the load balancer, there's some kind of dependency that terraform doesn't quite get)
+N.B. we need to ensure we explicitly target the `networking` module, otherwise we get an error when creating the load 
+balancer, as there's some kind of dependency that Terraform doesn't quite get.
+
+```terraform apply -target="module.networking" -target="module.front_door" -target="module.application_secrets" -var="initial_create=true"```
 
-This will create the certificates, load balancer, cloudfront distribution, and some networking and other things needed for defining DNS records.
-It will also create some app roles and secrets which are necessary before creating the full app.
+(This will create the certificates, load balancer, cloudfront distribution, networking and other components which are 
+the minimum required for defining the DNS records that DLUHC will need to add. It will also create some app roles and 
+secrets; which are necessary before creating the full app).
 
 ### Get DNS Records set up
 
-Ask DLUHC to set up DNS records required.
+Ask DLUHC to add the DNS records for the new environment.
 
-This will be;
+There will be four records, which will be;
```

Comment:
You could maybe add an example of each record here to make things super clear? Not necessary though.

Reply:
Added 👍